Updated DataDog Metrics screenshots (#5344)
Particular/docs.particular.net: samples/logging/datadog/sample.md

snippet: datadog-enable-nsb-metrics

## Dashboard

### Available Metrics

The following metrics are available across all endpoints. Use the tagged values `endpoint`, `messagetype_name`, and `messagetype_fullname` to aggregate and filter a metric.

![metrics list](metrics-list.png)

### Sample Dashboard

This dashboard replicates a lot of the data displayed by Service Pulse. But DataDog affords you a much longer history to draw from.

![dashboard](dashboard.png)

### Detailed Timeseries Example

This is a zoomed in example of a single Timeseries component. It displays the message throughput for the entire system, with each bar colour-coded by endpoint name.
Hovering over a section shows you details of the related endpoint.

e.g. `sum:nservicebus.processed{$dashboard_environment} by {endpoint}.as_rate().rollup(sum, 1)`

![throughput by endpoint](throughput-by-endpoint.png)

### Timeseries configuration

> You can pinpoint a specific endpoint or message type for the metric in the `from` setting, and then optionally breakdown the metric by another facet. 

Reviewer: ```suggestion
A specific endpoint or message type for the metric can be pinpointed in the `from` setting, and then optionally break down the metric by another facet. 
```

> In this example we display the `nservicebus.processed` metric for just the specified endpoint, and can then use the `sum by` to show individual sums by message type.

Reviewer: ```suggestion
In this example, the displayed `nservicebus.processed` metric for just the specified endpoint and can then use the `sum by` to show individual sums by message type.
```
